### DISCUSSION TITLE
1 passed, 3 warnings in 9.27s

### DISCUSSION THREAD

**Discussion by fricpto** [smartcontractkit/full-blockchain-solidity-course-py#1774] [Q&A] [answered]
when I compile my test folder it gives me this error : "1 passed, 3 warnings in 9.27s "
and this is my code:

```Python
from brownie import SimpleStorage, accounts

def test_deploy():
   
    account = accounts[0]
    
    simple_storage = SimpleStorage.deploy({"from": account})
    starting_value = simple_storage.retrieve()
    expected = 0
    assert starting_value == expected

    def test_updating_storage():
        
        account = accounts[0]
        simple_storage = SimpleStorage.deploy({"from": account})
       
        expected = 15
        simple_storage.store(
            expected, {"from": account}
        )  
  
        assert expected == simple_storage.retrieve()
```

<img width="722" alt="Screenshot 2022-10-22 195622" src="https://user-images.githubusercontent.com/100000316/200945346-2288ac63-f93f-4f04-86e3-a4ab911a8fef.png">

**Comment by cromewar** [comment, marked as answer]
Hello @fricpto 
This happens due to how the testing library make changes and brownie haven't updated those commands yet, however as you can see they are warning and not errors, so you can safely ignore them for now.
Let's hope Brownie maintainers update those commands on newer versions.

Cheers.

**Reply by fricpto** [reply]
thanks for your help.
